### PR TITLE
{perf}[intel/2015b] mympingpong 0.6.2 w/ Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires ~~#2098~~ and ~~#2099~~

also requires ~~https://github.com/hpcugent/easybuild-easyblocks/pull/729~~ and ~~#2102~~ for `mympingponganalysis` to work

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/775ca0d2776edae2aebb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/27e7c9c833320eb2843e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/29340b116068a27982fc for a full test report.

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10-timed-pingpong.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# force rebuilding everything, including patched files
+buildopts = '--force'
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%s/site-packages/mpi4py' % py_maj_min],
+}
+
+sanity_check_commands = [
+    ('python', '-c "from mpi4py.MPI import Comm; import sys; sys.exit((1, 0)[\'PingpongRS\' in dir(Comm)])"'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10-timed-pingpong.eb]: add a comment that this is supposed to do

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1_timed-pingpong.patch [hunk 2]
```diff
@@ -0,0 +1,332 @@
+--- src/MPI/Comm.pyx.orig	2010-04-29 15:50:19.332105000 +0200
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1_timed-pingpong.patch]: already there

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**